### DISCUSSION TITLE
Links — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#205] [Announcements]
https://mazhuang.org/links/

**Comment by zhuiwang** [comment]
i can leave a message?

**Comment by sunbufu** [comment]
马壮哥，能否在links页面添加上我的博客[https://sunbufu.github.io](https://sunbufu.github.io)，谢谢

**Comment by SWaiter** [comment]
很厉害

**Comment by ychost** [comment]
马哥，能加我的blog吗，[https://blog.sudoyc.com](https://blog.sudoyc.com)

**Comment by yizibi** [comment]
期待互友链：[一之笔](https://yizibi.github.io/)

**Comment by jueming0312** [comment]
[友链标题]：北京运维
[友链网址]：https://www.opsbj.com
[友链描述]：热爱生活、爱学习、专注Linux运维
博主好,想交换一个友链~~贵站友链我已经添加至首页~~希望博主给个位置~

**Comment by damon-dai** [comment]
> @wangenzhi
> [友链标题]：北京运维
> [友链网址]：https://www.opsbj.com
> [友链描述]：热爱生活、爱学习、专注Linux运维
> 博主好,想交换一个友链~~贵站友链我已经添加至首页~~希望博主给个位置~

123

**Comment by lxqxsyu** [comment]
加个友链

博客地址：https://dp2px.com
博客名称：水寒的博客

**Comment by mzlogin** [comment]
> @sunbufu
> 马壮哥，能否在links页面添加上我的博客[https://sunbufu.github.io](https://sunbufu.github.io)，谢谢

已加~

**Comment by mzlogin** [comment]
> @lxqxsyu
> 加个友链
> 
> 博客地址：https://dp2px.com
> 博客名称：水寒的博客

已加，域名有点意思~ :smile:

**Comment by lxqxsyu** [comment]
> @mzlogin
> > @lxqxsyu
> > 加个友链
> > 
> > 博客地址：https://dp2px.com
> > 博客名称：水寒的博客
> 
> 已加，域名有点意思~ :smile:

谢谢！明白人，哈哈

**Comment by gzwawj** [comment]
加个友链

- 博客地址：http://www.codeinfo.top
- 博客名称：聆听风声

**Comment by mzlogin** [comment]
> @gzwawj
> 加个友链
> 
> - 博客地址：http://www.codeinfo.top
> - 博客名称：聆听风声

已加~

**Comment by zhuiwang** [comment]
🐎哥，加个友链接吧 https://www.wangzhui.site
博客名称 MrLycorious‘s Website

**Comment by HwiLu** [comment]
码哥，fork了你的主题，期待互友链。:smile:

**Comment by mzlogin** [comment]
> @HwiLu
> 码哥，fork了你的主题，期待互友链。:smile:

已加~

**Comment by ghost** [comment]
fork了一下，因为不会改，就全部用了哈，不会侵权吧，期待互链：https://qinjisheng.top

**Comment by mzlogin** [comment]
> @qinjisheng2018
> fork了一下，因为不会改，就全部用了哈，不会侵权吧，期待互链：https://qinjisheng.top

没事儿～模板随意用，把我的博客文章、维基删除，个人资料换成你的就好。

你的链接我已加上，我的……你也加一下吧，你现在貌似链向自己了。

**Comment by ghost** [comment]
谢谢，那是我还弄好，好多东西都没弄呢，谢谢


| |
秦吉胜
|
|
qinjisheng2018@163.com
|
签名由网易邮箱大师定制
在2020年6月26日 09:49，Zhuang Ma<notifications@github.com> 写道：

@qinjisheng2018
fork了一下，因为不会改，就全部用了哈，不会侵权吧，期待互链：https://qinjisheng.top

没事儿～模板随意用，把我的博客文章、维基删除，个人资料换成你的就好。

你的链接我已加上，我的……你也加一下吧，你现在貌似链向自己了。

**Comment by PlanDoCheckAction** [comment]
博主，我 fork 了你的模板，非常感谢哈！我有个问题，怎么在自己写的 markdown 博客中用相对地址指向另一篇自己写的博客呀！我发现用相对地址去 link 的话，在网页站点内点击会跳到404页面。

**Comment by ghost** [comment]
不好意思，我也试了试，我的也不行，你问问码志吧，我博客的友情链接有他的联系方式，不好意思


| |
秦吉胜
|
|
qinjisheng2018@163.com
|
签名由网易邮箱大师定制
在2020年7月11日 20:54，涂斌<notifications@github.com> 写道：

博主，我 fork 了你的模板，非常感谢哈！我有个问题，怎么在自己写的 markdown 博客中用相对地址指向另一篇自己写的博客呀！我发现用相对地址去 link 的话，在网页站点内点击会跳到404页面。

**Comment by mzlogin** [comment]
> @PlanDoCheckAction
> 博主，我 fork 了你的模板，非常感谢哈！我有个问题，怎么在自己写的 markdown 博客中用相对地址指向另一篇自己写的博客呀！我发现用相对地址去 link 的话，在网页站点内点击会跳到404页面。

Jekyll 应该不会将文章文件的相对地址自动做转换，而且它在生成文章链接的时候文件名是按规则来生成的，比如我们用的是 `/年/月/日/名字/` 这样的。

我试了下以下几种写法，第一种不行，可以用后面两种：

```
[测试](./2011-05-25-export-data-from-excel-to-word-by-vba.md)

[测试](/2011/05/25/export-data-from-excel-to-word-by-vba/)

[测试](https://mazhuang.org/2011/05/25/export-data-from-excel-to-word-by-vba/)
```

**Comment by wangzi0218** [comment]
好奇怪，始终登录不上……
马壮哥，我想加一下我的博客 [见字如面]https://hiwannz.com

**Comment by wangzi0218** [comment]
惊呆了……
居然又登上了

**Comment by mzlogin** [comment]
> > @PlanDoCheckAction
> > 博主，我 fork 了你的模板，非常感谢哈！我有个问题，怎么在自己写的 markdown 博客中用相对地址指向另一篇自己写的博客呀！我发现用相对地址去 link 的话，在网页站点内点击会跳到404页面。
> 
> Jekyll 应该不会将文章文件的相对地址自动做转换，而且它在生成文章链接的时候文件名是按规则来生成的，比如我们用的是 `/年/月/日/名字/` 这样的。
> 
> 我试了下以下几种写法，第一种不行，可以用后面两种：
> 
> ```
> [测试](./2011-05-25-export-data-from-excel-to-word-by-vba.md)
> 
> [测试](/2011/05/25/export-data-from-excel-to-word-by-vba/)
> 
> [测试](https://mazhuang.org/2011/05/25/export-data-from-excel-to-word-by-vba/)
> ```

这个问题我找到了 Jekyll 官方的说明，用以下语法可以解决：

```
{% post_url 2010-07-21-name-of-post %}

{% post_url /subdir/2010-07-21-name-of-post %}

[Name of Link]({% post_url 2010-07-21-name-of-post %})
```

参考 <http://jekyllcn.com/docs/templates/#%E5%8D%9A%E6%96%87%E9%93%BE%E6%8E%A5post-url>

**Comment by mzlogin** [comment]
> @wangzi0218
> 好奇怪，始终登录不上……
> 马壮哥，我想加一下我的博客 [见字如面]https://hiwannz.com

加上了哈～

**Comment by jxnu-liguobin** [comment]
大佬加个链接吗（把博客迁移到fork你的模板上了）

梦境迷离 http://dreamylost.cn

**Comment by jxnu-liguobin** [comment]
本业访问量的显示好像有问题。。My Popular Repositories的时间好像不对

**Comment by mzlogin** [comment]
> @jxnu-liguobin
> 大佬加个链接吗（把博客迁移到fork你的模板上了）
> 
> 梦境迷离 http://dreamylost.cn

已添加

**Comment by mzlogin** [comment]
> @jxnu-liguobin
> 本业访问量的显示好像有问题。。My Popular Repositories的时间好像不对

My Popular Repositories 里的时间和内容是博客 build 时的快照，不是实时获取的。

本页访问量之前正常的，现在确实有点问题，不同页面显示的都是一样的，估计是 busuanzi 出问题了，等几天看能不能修正，不行就反馈反馈去。

**Comment by jxnu-liguobin** [comment]
原来是快照啊，实时是怕太慢吧

**Comment by river106** [comment]
大佬，加个友链吧
网站名称：river's blog
网站链接：https://www.river106.cn
网站描述：记录与分享

**Reply by mzlogin** [reply]
已添加哈～

**Comment by lanruixiang** [comment]
谢谢大佬，我 Fork 了你的模板，求求加个友链：<https://lanruixiang.github.io/>，感谢

**Reply by mzlogin** [reply]
好的加上啦～也麻烦你加下我的链接。

**Reply by lanruixiang** [reply]
好的谢谢啦

**Comment by Lch2010** [comment]
大佬，lanruixiang 教会我搭建这个博客，求求把我也放上去吧，感谢。

<https://lch2010.github.io/>，非常感谢

**Reply by mzlogin** [reply]
好的已添加。

**Comment by EasonHelloWord** [comment]
大佬大佬，我 fork 了你的模板，请求加个友链
https://www.easonjan.top/
非常感谢

**Reply by mzlogin** [reply]
好的已添加。
